src/current/_includes/releases/v23.1/v23.1.23.md

```diff
@@ -11,7 +11,7 @@ Release Date: June 20, 2024
     - There were multiple target tables with significant differences in row counts in one changefeed.
     - The changefeed target tables were large with many ranges.
     - The initial scan took a long time to complete (an hour or longer). [#123970][#123970] [#124759][#124759]
-- [Changefeeds]({% link v23.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all [replicas]({% link v23.1/architecture/glossary.md %}#replica), including [followers]({% link v23.1/architecture/replication-layer.md %}#raft), regardless of [leaseholder]({% link v23.1/architecture/glossary.md %}#leaseholder) placement. To disable this behavior, set the [cluster setting]({% link v23.1/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#124930][#124930]
+- Introduced the `changefeed.random_replica_selection.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}) that changes the behavior of changefeed planning work distribution. When `changefeed.random_replica_selection.enabled` is set to `true`, [changefeeds]({% link v23.1/change-data-capture-overview.md %}) will evenly distribute their work across all [replicas]({% link v23.1/architecture/glossary.md %}#replica), including [followers]({% link v23.1/architecture/replication-layer.md %}#raft), regardless of [leaseholder]({% link v23.1/architecture/glossary.md %}#leaseholder) placement. `changefeed.random_replica_selection.enabled` is disabled by default. [#124930][#124930]
```

Comment:
ah ok I see this cluster setting does already exist in my local v23.1.23 

so I guess we're not changing the name exactly but .. something else

there is also a public setting `changefeed.balance_range_distribution.enable`

this is rather confusing

Reply:
ah I see that v23.1.23 JUST BARELY came out [1], sorry for the noise

[1]: https://groups.google.com/a/cockroachlabs.com/g/releases/c/0ntb0-1DhQY/m/D-8nYQJqCQAJ

Reply:
No, totally ok Rich! This is my bad. This came about in Slack as a request and I thought it was going to be a really quick one. (Though I still should have filed a JIRA for this... but I will now!) Sorry you went through extra investigation because I did not provide the proper context.

Reply:
It was as a result of the wrong cluster setting name being listed in the PR description/commit that was then pulled into the release notes.

Reply:
no worries, idk anything about CDC other than "it exists" so it was interesting